### ISSUE TITLE
pip install montblanc fails because of new astropy dependency

### ISSUE THREAD

**Issue by o-smirnov** [ska-sa/montblanc#261]
I'm getting this when trying to ``pip install montblanc``:

```
Collecting astropy>=1.3.0 (from montblanc)
  Using cached https://files.pythonhosted.org/packages/21/7b/85cccff0b648752580664c479c377e6d3c7b018bc462e0417ed5ec8f46d4/astropy-3.0.5.tar.gz
    Complete output from command python setup.py egg_info:
    ERROR: Astropy requires Python 3.5 or later
```

What's the way out of?

**Comment by sjperkins** [comment]
astropy >= 3 is python 3 only. Quick fix is to put a python 2 constraint in
setup.py (example of how to do this is in codex-africanus setup.py).

I'll fix it up during the week.

**Comment by sjperkins** [comment]
Fixed by #264
